packages/lib/shared/components/tooltips/apr-tooltip/BaseAprTooltip.tsx

```diff
@@ -185,12 +187,12 @@ function BaseAprTooltip({
           tooltipText={merklIncentivesTooltipText}
         />
       )}
-      {hasSurplusIncentives && (
+      {isCowAmmPool(poolType) && (
         <TooltipAprItem
           {...basePopoverAprItemProps}
           displayValueFormatter={usedDisplayValueFormatter}
           title="Prevented LVR"
-          apr={surplusIncentivesAprDisplayed}
+          apr={hasSurplusIncentives ? surplusIncentivesAprDisplayed : bn(0)}
```

Comment:
Do we need `hasSurplusIncentives` here? I think `surplusIncentivesAprDisplayed` will just be zero right?

Reply:
You're right. Removed it.